composer.json

```diff
@@ -9,7 +9,8 @@
         }
     ],
     "require": {
-        "illuminate/database": "~5.6.0|~5.7.0"
+        "illuminate/database": "~5.6.0|~5.7.0",
+        "hanneskod/classtools": "~1.0"
```

Comment:
Initially hesitant to add a new dependency if we can use Reflection and such, but I'll wait to see your usage before making a judgment.

Reply:
I'm pretty sure we can do without this additional package if you prefer.